src/components/theme/useSystemTheme.ts

```diff
@@ -0,0 +1,20 @@
+import {useEffect, useState} from 'react';
+import {getDarkMediaMatch} from './getDarkMediaMatch';
+
+export function useSystemTheme(): 'light' | 'dark' | null {
+    const [theme, setTheme] = useState<'light' | 'dark' | null>(null);
+
+    useEffect(() => {
+        function onChange(event: MediaQueryListEvent) {
+            setTheme(event.matches ? 'dark' : 'light');
+        }
+
+        const matcher = getDarkMediaMatch();
+
+        matcher.addEventListener('change', onChange);
```

Comment:
Do we want support Safari 13? In Safari 13 this code will throw.

Reply:
It is not in our browserslist config

Reply:
But not in Readme. In most cases, this library works fine in Safari 13. With this changes, it will not work at all.

Reply:
What readme?..

Reply:
Does readme contain information about supported browsers? But it's not a main question.

Reply:
Revert `addListener` usage